docs/data/pages.ts

```diff
@@ -0,0 +1,31 @@
+import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
+
+const pages: MuiPage[] = [
+  {
+    pathname: '/toolpad',
+    title: 'Overview',
+    icon: 'DescriptionIcon',
+  },
+  {
+    pathname: '/toolpad/getting-started',
+    title: 'Getting Started',
+    icon: 'CreateIcon',
+    children: [
+      { pathname: '/toolpad/getting-started/setup', title: 'Setup' },
```

Comment:
I would expect this not to be required.

```suggestion
      { pathname: '/toolpad/getting-started/setup' },
```